arborist/server.go

```diff
@@ -395,8 +401,15 @@ func (server *Server) handleAuthRequest(w http.ResponseWriter, r *http.Request,
 		}
 	}
 	policies := []string{}
+	var username string
+	var clientID string
 	if info != nil {
 		policies = info.policies
+		username = info.username
+		clientID = info.clientID
+	} else {
+		username = authRequestJSON.User.UserId
+		clientID = ""
```

Comment:
we may eventually want this to also be something we can pass in... can we make a new ticket for the future to do that?

Reply:
https://ctds-planx.atlassian.net/browse/PXP-9106